turborepo-tests/integration/README.md

```diff
@@ -52,7 +52,7 @@ Setup
   $ . ${TESTDIR}/../../helpers/setup_integration_test.sh single_package
 
 Custom config
-  $ . ${TESTDIR}/../../helpers/replace_turbo_config.sh $(pwd) "my-custom-config.json"
+  $ . ${TESTDIR}/../../helpers/replace_turbo_json.sh $(pwd) "my-custom-config.json"
```

Comment:
This was an intentional rename - we have another helper that operates on the global turbo config, so I wanted to differentiate the turbo.json from the turbo config